android/app/build.gradle

```diff
@@ -44,7 +44,7 @@ android {
 
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
-        applicationId "com.hashirshoaeb.StarBook.star_book"
+        applicationId "com.hashirshoaeb.StarBook"
```

Comment:
It may effect on FIrebase because i set the application id `com.hashirshoeb.StarBook.star_book`, or not?

Reply:
I changed it. Thanks for pointing 🙌